src/components/PupilComponents/QuizEngineProvider/QuizEngineProvider.tsx

```diff
@@ -38,9 +38,11 @@ type QuestionState = {
 export type QuizEngineContextType = {
   currentQuestionData?: QuestionsArray[number];
   currentQuestionIndex: number;
+  currentQuestionDisplayIndex: number; // this excludes explanatory-text questions
```

Comment:
If you use JSDoc style for this (and other) comments it will appear as documentation in your editor to light the way for anyone using `useQuizEngineContext` down the line 

<img width="753" alt="Screenshot 2024-02-21 at 15 18 23" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/3d258842-39b7-48f3-ac30-9cd91fcff24f">